### ISSUE TITLE
Update to modern java standards?

### ISSUE THREAD

**Issue by indiealexh** [OpenSocial/activitystreams#22]
I'd love to see this reference impl be updated to use current java standards.

Is this project abandoned or just awaiting more activity?

**Comment by wilsncc** [comment]
@indiealexh I am wondering also what the status is of this project. Also interested to understand what you thing is outdated about this project?

**Comment by indiealexh** [comment]
Libraries are out dated.

It still uses Joda time instead of time introduced in Java 8.

Maven plugins out of date.

Java version is set to 1.5 for some of the modules.

There is some use of concurrency but Java has done a lot of work on those
apis since.

**Comment by wilsncc** [comment]
Yes, you’re unite quite right and I saw your PR. I emailed the contact email asking if the repo will be maintained or if it needs maintainers.

If it’s dead, would you be interested in forking it based on your PR?

**Comment by indiealexh** [comment]
I did start a project, for a Java based relay which I am hoping to use this
for.

So I'll see what I can do to maintain it further.
